ci/nur/combine.py

```diff
@@ -160,7 +159,7 @@ def setup_combined() -> None:
         write_json_file(dict(repos={}), manifest_path)
 
     manifest_lib = "lib"
-    copy_tree(str(ROOT.joinpath("lib")), manifest_lib, preserve_symlinks=1)
+    shutil.copytree(str(ROOT.joinpath("lib")), manifest_lib, preserve_symlinks=1)
```

Comment:
Unsure if we need this 
``` 
nur/combine.py:53: error: Unexpected keyword argument "preserve_symlinks" for "copytree"  [call-arg]
/nix/store/ryp4vp6rs3d1gk0mgj6x8bncavsp4wqa-python3.12-mypy-1.10.1/lib/python3.12/site-packages/mypy/typeshed/stdlib/shutil.pyi:64: note: "copytree" defined here
nur/combine.py:162: error: Unexpected keyword argument "preserve_symlinks" for "copytree"  [call-arg]
/nix/store/ryp4vp6rs3d1gk0mgj6x8bncavsp4wqa-python3.12-mypy-1.10.1/lib/python3.12/site-packages/mypy/typeshed/stdlib/shutil.pyi:64: note: "copytree" defined here
Found 2 errors in 1 file (checked 11 source files)
```

Reply:
```suggestion
    shutil.copytree(str(ROOT.joinpath("lib")), manifest_lib, ignore_dangling_symlinks=True)
```

Reply:
Huh… it _seemed_ like it was running okay. Looks like the correct argument for the `shutil` version is just `symlinks=True`. I'll fix that.

Reply:
Should be fixed now.